xml/System/DateOnly.xml

```diff
@@ -411,8 +413,8 @@
         <Parameter Name="dayNumber" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="dayNumber">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="dayNumber">The number of days since January 1, 0001 in the Proleptic Gregorian calendar.</param>
+        <summary>Creates a new instance of the DateOnly structure to the specified number of days.</summary>
         <returns>To be added.</returns>
```

Comment:
`to the specified number of days` sounds weird. Shouldn't it be `for the specified number of days`?:

```suggestion
        <summary>Creates a new instance of the <see cref="T:System.DateOnly" /> structure to the specified number of days.</summary>
        <returns>A <see cref="T:System.DateOnly" /> structure instance to the specified number of days.</returns>
```